src/ker_horner_allw_loop_constexpr.h

```diff
@@ -1,234 +1,235 @@
-// Code generated by gen_all_horner_C_code.m in finufft/devel
-// Authors: Alex Barnett & Ludvig af Klinteberg.
-// (C) The Simons Foundation, Inc.
+// Header of static arrays of monomial coeffs of spreading kernel function in each
+// fine-grid interval. Generated by gen_all_horner_cpp_header.m in finufft/devel
+// Authors: Alex Barnett, Ludvig af Klinteberg, Marco Barbone & Libin Lu.
+// (C) 2018--2024 The Simons Foundation, Inc.
 #include <array>
-template<uint8_t w> constexpr auto nc200() noexcept { return w + 2 + (w <= 8); }
+
+template<uint8_t w> constexpr auto nc200() noexcept {
+  constexpr uint8_t ncs[] = {4, 6, 7, 8, 9, 10, 10, 11, 12, 13, 14, 15, 16, 17, 18, };
+  return ncs[w-2];
+}
 
 template<class T, uint8_t w>
 constexpr std::array<std::array<T, w>, nc200<w>()> get_horner_coeffs_200() noexcept {
     constexpr auto nc = nc200<w>();
-    if constexpr (w == 2) {
+    if constexpr (w==2) {
```

Comment:
spaces, the formatter will auto add them if run before commit. Please let me know if devnotes.rst or contributing.md are not easy to follow.